apps/ui/src/views/SpaceUser/Statement.vue

```diff
@@ -88,7 +88,7 @@ watchEffect(() =>
         <UiMarkdown
           v-if="statement.statement"
           class="text-skin-heading max-w-[592px]"
-          :body="statement.statement"
+          :body="stripHtmlTags(statement.statement)"
```

Comment:
We should add this on Preview as well, it still display HTML tags
<img width="666" alt="Untitled 7" src="https://github.com/user-attachments/assets/846a94a5-a60f-4303-a56c-a0403a7e76f7">